cmd/quickstart.go

```diff
@@ -277,6 +285,62 @@ func quickstartExec(ctx context.Context, flags QuickstartFlags) error {
 	return nil
 }
 
+func retryWithSampleSpec(ctx context.Context, workflowFile *workflow.Workflow, initialTarget, outDir string, skipCompile bool) (bool, error) {
+	retrySampleSpec := true
+	_, err := charm.NewForm(huh.NewForm(
+		huh.NewGroup(
+			huh.NewSelect[bool]().
+				Title("\n\nYour OpenAPI spec seems to have some validation issues.\nWould you like to retry with a sample spec?\n").
+				Options(
+					huh.NewOption("Yes", true),
+					huh.NewOption("No", false),
+				).
+				Value(&retrySampleSpec),
+		),
+	)).ExecuteForm()
+	if err != nil {
+		return false, err
+	}
+
+	if !retrySampleSpec {
+		return false, nil
+	}
+
+	absSchemaPath := filepath.Join(outDir, "openapi.yaml")
+	if err := os.WriteFile(absSchemaPath, []byte(sampleSpec), 0o644); err != nil {
+		return true, errors.Wrapf(err, "failed to write sample OpenAPI spec")
+	}
+
+	workflowFile.Sources[workflowFile.Targets[initialTarget].Source].Inputs[0].Location = "openapi.yaml"
```

Comment:
I feel it's a bit strange to actually override the workflow file? Maybe not?

Reply:
We have too to get it to generate. I think we should otherwise their SDK still is in a failed state.